js/components/portal-dashboard/multiple-choice-answer.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from "react";
+import { Map } from "immutable";
+import Choice from "./choice";
+
+import css from "../../../css/portal-dashboard/multiple-choice-answer.less";
+
+const CORRECT_ICON = "icomoon-checkmark " + css.correct;
+const INCORRECT_ICON = "icomoon-cross " + css.incorrect;
+const SELECTED_ICON = "icomoon-checkmark2";
+
+interface IProps {
+  answer: Map<any, any>;
+  question: Map<any, any>;
+  showFullAnswer: boolean;
+}
+
+export default class MultipleChoiceAnswer extends PureComponent<IProps> {
+  renderIcon() {
+    const { answer, question } = this.props;
+    let icon;
+    if (!question.get("scored")) {
+      // Undefined as there's no correct or incorrect choice defined.
+      icon = SELECTED_ICON;
+    } else {
+      icon = answer.get("correct") ? CORRECT_ICON : INCORRECT_ICON;
+    }
```

Comment:
I'd probably shorten this with a ternary.

Reply:
Something like:
const icon = question.get("scored") ? answer.get("correct") ? CORRECT_ICON : INCORRECT_ICON : SELECTED_ICON;

Reply:
A triple?
icon = !question.get("scored") ? SELECTED_ICON : answer.get("correct") ? CORRECT_ICON : INCORRECT_ICON;

Reply:
Sorry that was a typo.

Reply:
Meant to write:
`const icon = question.get("scored") ? SELECTED_ICON : answer.get("correct") ? CORRECT_ICON : INCORRECT_ICON;`

But if that is too unreadable, then just leave it.  Or this:
```
const icon = question.get("scored") 
                     ? SELECTED_ICON 
                     : answer.get("correct") ? CORRECT_ICON : INCORRECT_ICON;
```